### PR TITLE
[don't merge]CI:just have a try, if kind will upload images or not

### PR DESCRIPTION
**Pull request by SamYuan1990**
Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -24,8 +24,15 @@ jobs:
...
-            PROMETHEUS_ENABLE: "true"
-            GRAFANA_ENABLE: "false"
+            
+      - name: Checkout kube-operator repo
+        uses: actions/checkout@v3
+        with:
+          repository: prometheus-operator/kube-prometheus
+          path: kube-prometheus
+      - name: deploy kube operator
+        run:  ls && kubectl create -f manifests/setup && until kubectl get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done && kubectl create -f manifests/
```
Comment (marceloamaral) [inline .github/workflows/integration_test.yml:34]: You need to check if the containers were created!

For example, you can add after `kubectl create -f manifests/`:
`; kubectl wait --for=condition=Ready pod --all -n monitoring --timeout 12m`

**Review by marceloamaral — CHANGES_REQUESTED** [review]

File: .github/workflows/integration_test.yml (hunk 1 above)
Comment (SamYuan1990) [reply .github/workflows/integration_test.yml:34]: you mean, if not, kepler pod will success but namespace is monitoring may not ready?

Comment (marceloamaral) [reply .github/workflows/integration_test.yml:34]: prometheus operator containers will not be ready

if we don't care if the prometheus operator containers are running we don't need to deploy it....

Comment (SamYuan1990) [reply .github/workflows/integration_test.yml:34]: will if we have integration test case. them will test rest api for kepler. and should we need test output from promethes?

Comment (marceloamaral) [reply .github/workflows/integration_test.yml:34]: Yes, that is why we need to make sure that the prometheus operator containers are running
As I am suggesting to do :)

Comment (marceloamaral) [reply .github/workflows/integration_test.yml:34]: Note that, prometheus operator containers are created in the monitoring namespace, then we need to verify if those container a Ready

Comment (SamYuan1990) [reply .github/workflows/integration_test.yml:34]: are we force kepler bind with prometheus?

> Yes, that is why we need to make sure that the prometheus operator containers are running As I am suggesting to do :)

Comment (marceloamaral) [reply .github/workflows/integration_test.yml:34]: Did not understand your question...

Prometheus operator must successfully run

**Pull request closed without merging**